src/components/Episode/styles.scss

```
@@ -1,7 +1,31 @@
 .wrapper {
-	font-size: 1em; //placeholder styling
-
+	margin-top: 20vh;
+	position: absolute;
+	
 	&[aria-hidden="true"]{
 		opacity: 0;
 	}
-}
\ No newline at end of file
+}
+
+	.content {
+		padding: 10%; 
+		font-size: 1.3rem;
+	}
+
+		.episode {
+			margin-bottom: 1.67em;
+			font-size: 1em;
+			font-family: var(--font_family__heading);
+		}
+
+		.title {
+			font-size: 1.7em;
+			font-family: var(--font_family__heading);
+		}
+
+		.body {
+			margin-top: 1.5em;
+			line-height: 1.5em;
```

```suggestion
			line-height: 1.5;
```
This is correct, but Edge still fucking shits bricks if you provide a `line-height` that has units - luckily, `1.5` is `1.5em` so we're good to continue. Just an annoying quirk to memorise I'm afraid - although this may change now that Edge is a Chromium build.